content/blog/2023-06-27-the-life-cycle-of-an-etc-transaction/index.md

```diff
@@ -0,0 +1,96 @@
+2. The wallet app sends the transaction to a remote node of the ETC network that it uses to process transactions
+3. The remote node receives the transaction, confirms that the sending address has a sufficient balance, and that the transaction is correctly signed with the private key
+4. The transaction is sent to the rest of the network
+5. The rest of the network also verifies and processes the transaction and includes it in the blockchain forever
+
+## Dapp or Web3 Transaction
+
+![Dapp or web3 transaction.](./2.png)
+
+A dapp or web3 transaction uses more steps and it calls smart contracts on the blockchain to execute more complex operations. The way it works is as follows:
+
+1. The user goes to a dapp or web3 website and connects his or her wallet to that application
+2. The user initiates a transaction, for example a DeFi trade or buying or selling an NFT, and presses send on the dapp or web3 application
+3. The dapp or web3 application generates the transaction, which is usually to interact with a smart contract on the blockchain, and feeds it back to the user’s wallet
+4. The user’s wallet then asks the user to accept and sign the transaction with his or her private key and sends it to a remote node of the ETC network that it uses to process transactions
+5. The remote node receives the transaction and verifies that it is correct and signed
+6. The transaction is sent to the rest of the network
+7. The rest of the network also verifies and processes the transaction and includes it in the blockchain forever
+
+## What Do Miners Do?
+
+![The work of miners.](./3.png)
+
+When we say above that the network “verifies and processes the transaction and includes it in the blockchain forever” what we mean is that the subgroup of the network nodes which are the miners get the transactions and include them in blocks. The way they do this is as follows:
+
+1. Mempool: All the new transactions are accumulated in a pool of unprocessed transactions called the “mempool” which stands for “memory pool”
+2. Proof of Work: Miners grab as many transactions as they can wich pay the highest fees (this is why it is important to send transactions with the correct gas fees), create a batch, and then hash them cryptographically in a process called proof of work
+3. Block: After doing a lot of computational work and spending a lot of electricity, they stamp the batch of transactions with a cryptographic hash and create what is called a block of transactions
+4. Network: When they have the block ready, then they immediately send it to the rest of the network for verification and full replication in all nodes of the system
+
+## How Can I Know if a Transaction Is Included in the Chain?
+
+![Transaction on a block explorer.](./4.png)
+
+When any transaction sent by a user goes through all the steps mentioned above, then it will be recorded in the blockchain forever, as said before.
```

Comment:
Line 72: Reorder wording for readability
```suggestion
When any transaction sent by a user goes through all the steps mentioned above, it will then be recorded in the blockchain forever, as said before.
```